mods.html

```diff
@@ -4,7 +4,7 @@ <h1>Modifications to the C++ Standard Library</h1>
   <cxx-section id="mods.general">
     <h1>General</h1>
     <p>
-      Implementations that conform to this technical specification shall
+      Implementations that conform to this document shall
```

Comment:
I'm not sure what it means to conform to this document when we've already stated that it's non-normative, but I don't have a better suggestion right now. And I don't think it's a big deal to leave it like this.

Reply:
Happily, ISO has separately asked to remove this sentence entirely (with an amusing explanation), so this problem will go away in the next commit.

Reply:
ISO has separately requested that we remove the statement that we're non-normative (with an amusing explanation), so hopefully that resolves this tension.